src/Geta.Optimizely.Sitemaps/Services/NullUriAugmenterService.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using EPiServer.Core;
+using EPiServer.ServiceLocation;
+
+namespace Geta.Optimizely.Sitemaps.Services
+{
+    [ServiceConfiguration(typeof(IUriAugmenterService))]
```

Comment:
Do not use `ServiceConfiguration` attribute - it is legacy from .NET Framework projects and should not be used.
Instead register services in `ServiceCollectionExtensions`.

Reply:
No problem.